client/driver/executor/executor.go

```diff
@@ -0,0 +1,217 @@
+package executor
+
+import (
+	"fmt"
+	"log"
+	"os"
+	"os/exec"
+	"path/filepath"
+	"runtime"
+	"sync"
+	"syscall"
+	"time"
+
+	cgroupConfig "github.com/opencontainers/runc/libcontainer/configs"
+
+	"github.com/hashicorp/nomad/client/allocdir"
+	"github.com/hashicorp/nomad/client/driver/env"
+	"github.com/hashicorp/nomad/nomad/structs"
+)
+
+// ExecutorContext is a wrapper to hold context to configure the command user
```

Comment:
Can remove "is a wrapper to" on all of these.

`// ExecutorContexts holds information to configure and isolate the user command`